mod-arch-kubeflow/style/MUI-theme.scss

```
@@ -728,12 +738,57 @@
+.mui-theme .pf-v6-c-input-group .pf-v6-c-input-group__item .pf-v6-c-button.pf-m-control {
+  --pf-v6-c-button--BorderColor: transparent;
+  --pf-v6-c-button--hover--BorderColor: transparent;
+  --pf-v6-c-button--m-control--BorderColor: transparent;
+  --pf-v6-c-button--m-control--hover--BorderColor: transparent;
+  --pf-v6-c-button--m-control--m-clicked--BorderColor: transparent;
+  --pf-v6-c-button--PaddingBlockStart: 0;
+  --pf-v6-c-button--PaddingBlockEnd: 0;
+  --pf-v6-c-button--PaddingInlineStart: 0;
+  --pf-v6-c-button--PaddingInlineEnd: 0;
+  --pf-v6-c-button--m-control--BorderRadius: 50%;
+  width: var(--mui-input-group-control-button--size);
```

Stylelint (17.3.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)